.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
I'd rather not have dependabot configured here, could you remove it please?

Reply:
Ok, then I will not contribute anymore updates which are easily automated by dependabot.